accounts-db/src/accounts_hash.rs

```diff
@@ -1160,16 +1255,15 @@ impl AccountsHasher<'_> {
         let binner = PubkeyBinCalculator24::new(bins);
 
         // working_set hold the lowest items for each slot_group sorted by pubkey descending (min_key is the last)
-        let (mut working_set, max_inclusive_num_pubkeys) = Self::initialize_dedup_working_set(
+        let (mut working_set, _max_inclusive_num_pubkeys) = Self::initialize_dedup_working_set(
```

Comment:
max_inclusive_num_pubkeys is an estimate of the upper bound for the hash file size. It is only required when we use mmap - creating an mmap requires to specify the initial size, which could be over allocated too. After switching to file writer, we don't need this any more.